src/Popover/Popover.js

```diff
@@ -79,6 +85,12 @@ class Popover extends Component {
             onClickFunctions = chain(this.triggerBody, control.props.onClick);
         }
 
+        let newPopperProps = !!popperProps ? popperProps : {};
+        // eslint-disable-next-line no-undefined
+        if (newPopperProps.id === undefined || newPopperProps.id === null) {
+            Object.assign(newPopperProps, { id: this.ariaControls });
+        }
```

Comment:
I wonder if this could be simplified and avoid the safe checking by:
1) Setting `defaultProps` for `popperProps = {}`
2) Just assigning an `id` inline?
```js
const id = popperProps.id || this.ariaControls;
...
popperProps={{ ...popperProps, id }}
```